controller/src/main/java/org/jboss/as/controller/capability/RuntimeCapability.java

```diff
@@ -391,9 +462,7 @@ private Builder(String baseName, boolean dynamic, T runtimeAPI) {
          * @param requirements the capability names
          * @return the builder
          *
-         * @deprecated nothing is currently done with this data and unless a true use is implemented this method may be removed
          */
-        @Deprecated
```

Comment:
IIRC the use case we were thinking about for this went away, so it should remain deprecated